js/RepoLabels.js

```diff
@@ -5,14 +5,18 @@ let backends = {};
 export default class RepoLabels extends Array {
 	constructor(name, options) {
 		super();
-		this.name = name;
-		this.backend = backends[name] ?? new Backend(`https://github.com/${name}/labels`, options);
-
-		this.backend.load().then(d => {
-			for (let label of d) {
-				this.push(label);
-			}
-		});
+
+		if (options) {
```

Comment:
We need this to ensure we create an actual backends and do not perform any array operation, such as `map`, `splice`, etc., while working with an instance of `RepoLabels` class.

Reply:
Do not add GitHub comments that explain what code does — that’s what code comments are for!